### PR TITLE
Do we need `npm run compile` twice?

### PR DESCRIPTION
**Pull request by mccalluc**
(I'll default to listing you both as reviewers going forward, and will generally wait for a green check mark from Pete, and no objections from Fritz. Let me know if some other approach would be better.)

### CODE REVIEW

**Comment by flekschas** [comment]
> Do we need `npm run compile` twice?
Can you point out where the first compile is happening?

Without compiling, where is `dist.zip` coming from?

**Comment by pkerpedjiev** [comment]
I second Fritz's question.

**Comment by mccalluc** [comment]
@pkerpedjiev @flekschas : There had been two in travis.yml, and one is now in test.sh. I just added a check there to make sure dist.zip exists afterwards. I don't think running the tests would remove it, so I don't think it needs to be generated again.

**Comment by mccalluc** [comment]
Renamed; thanks! Give me a green checkmark, and I'll merge it.

File: travis_build_test.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ end eslint
 
 start compile
 npm run compile
+[ -e dist.zip ] || die 'Missing dist.zip'
```
Comment (pkerpedjiev) [inline travis_build_test.sh]: Maybe say `Missing dist.zip, please check that it was produced by npm compile in build.sh`

Comment (flekschas) [reply travis_build_test.sh]: I 😍
@mccalluc other than that please merge

**Pull request merged**